config/rewrite.toml

```diff
@@ -1,10 +1,5 @@
-# Rewrite rules.
-# - all rules are applied to each record
-# - if rule mathches, metric path is rewriten
-# - if copy is true, new metric is created
-
 [[rewrite]]
```

Comment:
Please comment this block as this is default config.
We don't want to ship default config with rewrites.

Please add the next example which demonstrating how to use regex capturing group
[[rewrite]]
    from = '^sys\.([A-Za-z0-9_-]+)\.(.*)'
    to = 'sys.$2.$1'
    copy = true

Reply:
Default config has rewrites unset.

I may add this in the next iteration of the documentation. I would keep it simple for now.